pkg/types/spec/errors.go

```diff
@@ -166,6 +168,20 @@ func ErrorConfigGreaterThanOtherConfig(tooBigKey string, tooBigVal interface{},
 	})
 }
 
+func ErrorMinReplicasLessThan(min int32, limit int32) error {
```

Comment:
These errors can be deleted if we move the validation to the configreader as discussed